production/helm/loki/templates/statefulset.yaml

```diff
@@ -51,6 +51,9 @@ spec:
             - "-{{ $key }}={{ $value }}"
           {{- end }}
           volumeMounts:
+{{- if .Values.extraVolumeMounts }}
```

Comment:
Indent to be same level as config volume mount using `nindent` to match how everything else is indented in the file.